test/functional/wallet_listdescriptors.py

```diff
@@ -103,7 +103,7 @@ def run_test(self):
             'desc': descsum_create('wpkh(' + xpub_acc + ')'),
             'timestamp': 1296688602,
         }])
-        assert_raises_rpc_error(-4, 'Can\'t get descriptor string', watch_only_wallet.listdescriptors, True)
+        assert_equal(watch_only_wallet.listdescriptors(True), watch_only_wallet.listdescriptors(False))
```

Comment:
Is this desired behaviour? Should we check that wallet doesn't have `disable_private_keys` enabled and return "wallet doesn't contain private keys" error?